assin.php

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title></title>
+</head>
+<body>
+<?php
+$f_name =  $_POST["fname"];       //store first name via post method
```

Comment:
Add a check condition at the top level for checking if the method of form submission.
Also I can't see any check for User will input only alphabets, I believe we can add digits also in the field of first & last name. Kindly add that also.